site/content/entry/yuzu-progress-report-mar-2022/index.md

```diff
@@ -0,0 +1,282 @@
+	"./invisbug.png"
+	"./invisfix.png"
+>}}
+
+&nbsp;
+
+{{< single-title-imgs-compare
+	"Super Mario Galaxy"
+	"./smgbug.png"
+	"./smgfix.png"
+>}}
+
+`Super Mario Galaxy` uses a combination of depth buffer and stencil buffer to “see” the objects behind the star pointer used to shoot star bits.
+That’s right, a texture determines game physics.
+
+{{< gh-hovercard "8030" "Properly handling that S8D24 < > ABGR8 texture conversion" >}} solves wrong collisions with the poor star bits.
+{{< gh-hovercard "8036" "A later fix solves issues with Nvidia drivers" >}}.
+
+{{< single-title-imgs
+	"No star bits for you until you finish your homework (Super Mario Galaxy)"
+	"./texturebug.mp4"
+	"./texturefix.mp4"
+>}}
+
+A shader miscompilation (a bad assumption on yuzu’s part, oops) caused interesting black holes in Sunshine.
+{{< gh-hovercard "8038" "Fixing the oopsie" >}} gives Delfino the intended “shading”, and most likely silently solves issues in other games too.
+
+{{< single-title-imgs-compare
+	"Super Mario Sunshine"
+	"./miscompbug.png"
+	"./miscompfix.png"
+>}}
+
+And finally, {{< gh-hovercard "8074" "fixing a bottleneck in the buffer cache" >}} eliminates slowdowns and helps improve performance by up to approximately 4.5x in Galaxy.
+Again, this could have a positive impact on other games, but don’t expect it to be in such a dramatic way.
+
+[Merry](https://github.com/merryhime) updated [dynarmic](https://github.com/merryhime/dynarmic) to the latest version to provide better support for `Super Mario Galaxy`, {{< gh-hovercard "8054" "adding support for accelerated SHA256 CPU instructions" >}}, among other changes.
+This avoids softlocking Galaxy at the final level.
+
+{{< single-title-imgs
+	"This toad sure likes to talk a lot (Super Mario Galaxy)"
+	"./creditbug.mp4"
+	"./creditfix.mp4"
+>}}
+
+## POYO!
+
+`Kirby and the Forgotten Land` turned out to be one of the best platformers of recent times, and also quite a challenge for our devs.
+
+epicboy jumped in to keep the ball rolling and started by solving the geometry pop-ins that are very noticeable in the first gameplay scenes.
+
+{{< single-title-imgs
+	"That tree just pops! (Kirby and the Forgotten Land)"
+	"./popbug.mp4"
+	"./popfix.mp4"
+>}}
+
+This was achieved by {{< gh-hovercard "8023" "improving the accuracy of the constant buffer uploads" >}}, and has the added benefit of also fixing rendering issues in `Monster Hunter Ries` and `Diablo® II: Resurrected™`.
+
+{{< imgs
+	"./rise.png| Such a pretty intro (MONSTER HUNTER RISE)"
+  >}}
+
+&nbsp;
+
+{{< imgs
+	"./diablo2.png| A real classic (Diablo® II: Resurrected™)"
+  >}}
+
+As you can see, Diablo II has some UI rendering issues, but at least every character has access to all their limbs now!
+
+Another issue that was quickly pointed out, is that Nvidia hardware would eat VRAM like crazy, to the point of filling up even high end hardware in minutes.
+
+{{< imgs
+	"./kirbyvram.png| PC requirements: RTX 3090? Not on our watch!"
```

Comment:
Vram PR may be closed bc of regressions it causes in other titles. Can you advise users to check out the mods for this game? Particularly the disable dynamic res mod.